lib/Dialect/CKKS/IR/CKKSDialect.h

```diff
@@ -0,0 +1,12 @@
+#ifndef THIRD_PARTY_HEIR_LIB_DIALECT_CKKS_IR_CKKSDIALECT_H_
```

Comment:
could you check the header guards here and elsewhere? Internally they should all be LIB_ (I know this goes against the internal automatic style, and I'll post a better copybara transform for them, but for now let's make them all uniform so the pattern will be applied to everything)

Reply:
Done.

Reply:
Nice, I think Jeremy's pattern matched on LIB and so missed the ones in the tests folder. I'll add them ot the issue I had filed.

Reply:
Filed a PR to fix it and add a pre-commit guard.